server/rooms.ts

```diff
@@ -327,7 +327,7 @@ export abstract class BasicRoom {
 	/**
 	 * Gets the group symbol of a user in the room.
 	 */
-	getAuth(user: User): GroupSymbol {
+	getAuth(user: {id: ID, group?: GroupSymbol | undefined} | User): GroupSymbol {
```

Comment:
```suggestion
	getAuth(user: {id: ID, group: GroupSymbol} | User): GroupSymbol {
```

ts-eslint complains about the weirdest things, I thought it would catch this double `undefined`.

Anyway, this does not seem like the correct place to assert.